x/evidence/go.mod

```diff
@@ -13,14 +13,14 @@ require (
 	cosmossdk.io/store v1.0.0-rc.0
 	github.com/cometbft/cometbft v0.38.0
 	github.com/cosmos/cosmos-proto v1.0.0-beta.3
-	github.com/cosmos/cosmos-sdk v0.46.0-beta2.0.20230713190037-6a0ab4fd167f
+	github.com/cosmos/cosmos-sdk v0.47.5
```

Comment:
we should push this to the latest version of the releases?

cc @julienrbrt

Reply:
We should either use a pseudo version from main (v0.46.0-beta2.xxxx) or a fake version (https://github.com/cosmos/cosmos-sdk/pull/17776#discussion_r1342100411) and replace the SDK.

Definitely not a released version on main.